Assets/Project/Scripts/GamePlayScene/GamePlayDirector.cs

```diff
@@ -242,8 +239,8 @@ private void GameFail()
         {
             if (OnFail != null) OnFail();
             EndProcess();
+            _resultText.GetComponent<Text>().text = LanguageUtility.GetText(ETextIndex.GameFailure);
```

Comment:
別にどっちでも行けるけど、想像してた使い方はこんな感じだった。
```suggestion
            _resultText.GetComponent<Text>().TextIndex = ETextIndex.GameFailure;
```

Reply:
これはいろんなところにあるね

Reply:
いや、ごめん、違った。どっちでも行けるじゃない。 `text` だけ変えると `TextIndex` は変えないので間違ってる。なので悪いけど修正してください。

Reply:
`TextIndex`を`set`したときに`LanguageUtility.GetText()`呼ばれてるのか。
理解した！